cypress/e2e/helpers/answerSurvey.ts

```diff
@@ -0,0 +1,84 @@
+import moment from "moment";
+import "moment/min/locales";
+
+import * as assignSurvey from "../helpers/adminHelpers/assignSurvey";
+
+export const TEXT_DATE: string = "p.MuiTypography-body1.css-xhbppr";
+export const BUTTON_CLOSE: string = "button";
+export const INPUT_VALUE_LBS: string = "input.MuiInputBase-input";
+export const TEXT_LBS: string = "p.MuiTypography-root";
+export const IMG_PROGRESS_BAR_50: string =
+  'span[role="progressbar"][aria-valuenow="50"]';
+export const IMG_PROGRESS_BAR_100: string =
+  'span[role="progressbar"][aria-valuenow="100"]';
+
+export const TEXT_VALUE_PROGRESS_BAR: string = "p.MuiTypography-root";
+export const BUTTON_NEXT: string = "button.MuiButton-containedPrimary";
+export const SNACK_BAR_ALERT_COMPLETED: string =
+  '[data-testid="SNACK_BAR_ALERT"]';
+
+export const verifySurveyCard = () => {
+  cy.contains("h6", "SER-ISD-001 - Body Weight");
+  cy.contains("h6", "OPEN");
+
+  const currentDate = moment().format("MMM D");
+  const expectedText = `Once on ${currentDate} at ${assignSurvey.calculatedTime}`;
+  cy.log(expectedText);
+  cy.contains("h6", expectedText);
```

Comment:
Here we could create a function to return the correct string

const SCHEDULE_ONCE_TEXT = (date: string, time: string) => `Once on ${date} at ${hour}`
```suggestion
  cy.contains("h6", SCHEDULE_ONCE_TEXT(date, hour));
```